README.md

```diff
@@ -192,6 +192,46 @@ display.display(pd.DataFrame.from_records([["col 1": 3, "col 2": 5], ["col 1": 8
 
 [Swift's Python interop]: https://github.com/tensorflow/swift/blob/master/docs/PythonInteroperability.md
 
+## Inline plots
+
+You can display images using Swift too. 
+
+Prerequisites:
+
+* You need to install the BlueCryptor library using the `%install` directive. We use it for message signing.
+```swift
+%install '.package(url: "https://github.com/IBM-Swift/BlueCryptor.git", from: "1.0.28")' Cryptor
+```
+
+After doing this, run `%include "EnableJupyterDisplay.swift"` in your Swift notebook. Now you should be able to display images! (Currently only PNG format is supported. You also need to provide the image as a base64 String. Eventually we'd like to support other formats as well.)
```

Comment:
Suggestion, replace everything up to "For example:" with

````markdown
You can display images using Swift too. 

```swift
%install '.package(url: "https://github.com/IBM-Swift/BlueCryptor.git", from: "1.0.28")' Cryptor
%install '.package(url: "https://github.com/KarthikRIyer/swiftplot", from: "0.0.1")' SwiftPlot
%include "EnableJupyterDisplay.swift"
```

Now you should be able to display images! (Currently only PNG format is supported. You also need to provide the image as a base64 String. Eventually we'd like to support other formats as well.)

For example:
````

This is shorter, and its a single block of code that users can just paste into a cell and start using immediately without having to read carefully.